docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,130 @@
+
+### Digital clock
+
+This feature is an alternative to the analog clock on time pickers; the original proposal was mainly focused on mobile devices, whereas the new interface is designed for a desktop experience or better time precision.
+This new addition makes our date pickers even more versatile and user-friendly and is available as the default variant for desktops on both [TimePicker](https://mui.com/x/react-date-pickers/time-picker/) and [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/).
+
+<img src="/static/blog/mui-x-mid-v6-features/timepicker.gif" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Check out all the possibilities in [documentation](https://mui.com/x/react-date-pickers/digital-clock/), and let us know your thoughts!
+
+## Data Grid
+
+### Copy and Paste [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+
+After setting the prop `experimentalFeatures={{ clipboardPaste: true }}` you can copy and paste data directly from and to your Data Grid.
+
+This new feature is extremely versatile, and you can exchange data from other fields, other grids, and even directly from and to Microsoft Excel.
+
+This feature is integrated with the editing API, so pasted data can be persisted using the `processRowUpdate` prop to update your data source through your usual editing validation process.
+Furthermore, the callbacks `clipboardPasteStart` and `clipboardPasteEnd` are fired during the clipboard paste operation, which can be useful for extra customizations around the event.
+
+For more details on how to use clipboard copy and paste, check out the [feature's page](https://mui.com/x/react-data-grid/clipboard/).
+
+### Filter on column headers [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+This feature allows users to quickly filter the data without needing any additional menu and can be activated using the `unstable_headerFilters` prop.
+
+The new filter fields are displayed below the headers and are synchronized with the filter panel, but for those seeking more simplicity, disable the default filter panel using `disableColumnFilter` prop and set filters to use only the default operator.
+You can also customize the header filter cell for a specific column using the `renderHeaderFilter` method of `GridColDef`
+
+For more details on how to use and customize the filters, check out its [documentation page](https://mui.com/x/react-data-grid/filtering/header-filters/).
+
+### **Performance Enhancements: Smoother Scrolling and Expanding Rows**
+
+The Data Grid received significant performance improvements in MUI X v6, particularly concerning the data grid's scrolling efficiency and the expansion of children's rows.
+
+These performance upgrades are part of our ongoing commitment to provide end-users with a superior experience.
+We always strive to ensure that MUI X remains performant and efficient, particularly when dealing with large and complex data.
```

Comment:
@romgrk do you have ideas on how we could sell these improvements to the developers? I think it would be awesome to have some benchmark of before vs. after to get them excited, similar to https://www.prisma.io/blog/prisma-5-f66prwkjx72s.

Reply:
Scrolling: We did get interesting improvements in scrolling performance, but it's very tame. It was from 10-20% less CPU usage on each scroll event, which is nice but isn't really noticeable. Maybe for low-end devices it's more noticeable.

Filtering: I could benchmark master 4 months ago and now, though I'm not finished with the improvements. Let me know when you need this I'll get you the numbers.

Reply:
By the way, listing way to improve perf could be a good blog article topic :)

Reply:
Here is an approximate but representative benchmark for now vs before the filtering improvements:

| | Average (ms) | StdDev (ms) |
| --- | --: | --: |
| Current | `118.8` | `+- 4.8` |
| Previous | `811.8` | `+- 16.0` |

So we have improved by 85% for this benchmark. The benchmark is a one-column string filter on 100,000 rows of the commodity dataset.

Reply:
@romgrk wonder if it would be beneficial to add that 85% improvement table on the blog?

Reply:
I don't have a strong opinion, but I would finish the filtering improvements before announcing them.

Reply:
gotcha. wont add for now then.

Reply:
At a high level, I think the order of effectiveness of the content:

1. (current state) No number, no ways to back the claim. I wonder if the blog wouldn't be better without the h3, allowing readers to focus more of their time on the rest of the blog.
2. Link to the relevant PRs, this would be a start.
3. A number without clearly explaining the reproduction case: https://nextjs.org/blog/next-8#massive-build-time-memory-usage-reduction it feels strange to read though, I almost feel 2 is a stronger case.
4. A number with a clear reproduction case: https://nextjs.org/blog/next-7#compilation-speed, https://nextjs.org/blog/next-8#smaller-initial-html-size, great!

Is there a way we could link the relevant PRs in the blog post 🙌?

Reply:
There's a bunch of CTA in the blog already and don't want crown it with more CTA. Im leaning toward removing the h3.

Reply:
Ok removed